source/tasks/AwaitTask/AwaitTaskV6/waiter.ts

```diff
@@ -27,7 +34,7 @@ export class Waiter {
         const waiter = new ServerTaskWaiter(client, inputParameters.space);
 
         const taskIds: string[] = [];
-        const failedTaskIds: string[] = [];
+        const deploymentResults: DeploymentResult[] = [];
```

Comment:
I'd missed this in the earlier reviews. This isn't always a `DeploymentResult`, sometimes it's a `RunbookRunResult`.

Reply:
maybe rename to `TaskResult`?